### ISSUE TITLE
failed to import new osm by using import.sh

### ISSUE THREAD

**Issue by cswanghan** [a-b-street/abstreet#534]
Got an empty reply while trying to download google_transit_2018_18_08.zip, any work arounds?

**Comment by cswanghan** [comment]
![image](https://user-images.githubusercontent.com/699970/109113285-f9756a00-7776-11eb-97cf-151f6ad8fe83.png)

looks like they changed the file name;

**Comment by dabreegster** [comment]
Thanks for reporting! I can fix the link tomorrow. The workaround:

1) Edit `data/player/data.json` and add "us/seattle" to the "input" section. If you don't have this file yet, then you haven't done `cargo run --bin updater` before
2) Do `cargo run --bin updater`. It'll grab all of the input files for Seattle, including the copy of the GTFS stuff

But also, you don't need to build the Seattle map yourself unless you want to. If you're just trying to run, `cargo run --bin updater` will pull down the pre-built maps.

**Comment by cswanghan** [comment]
error: could not compile `geom`

then, I got an error while run ./import.sh --oneshot=new.osm.

I am using
 rustc 1.52.0-nightly (9b471a3f5 2021-02-19)

**Comment by dabreegster** [comment]
What's the error? We only support Rust stable, currently 1.50. https://a-b-street.github.io/docs/dev/index.html. You can use rustup to install both stable and nightly, if you want both.

But if you're just trying to import a map to try things out, two easier options:

1) Use the latest [binary release](https://github.com/dabreegster/abstreet/releases), which has the importer pre-built
2) Send the .geojson or clipping polygon, and I can do the import

We're going to make the import process easier in a few weeks, to hopefully lessen this hassle.

**Comment by dabreegster** [comment]
> Got an empty reply while trying to download google_transit_2018_18_08.zip, any work arounds?

Wait, going back to the original bug, how did this happen? https://github.com/a-b-street/abstreet/blob/fa4d46e64896f9c22f04a215b3bfe0f39f68eb4c/importer/src/seattle.rs#L52 fetches metro.kingcounty.gov/gtfs/google_transit.zip, which still exists. Where did `google_transit_2018_18_08.zip` come from?

**Comment by cswanghan** [comment]
Looks like I am not using the most updated version, I have pulled first from the master branch but somehow my local repo was not updated. Therefore, the import.sh I used lead me to [metro.kingcounty.gov/gtfs/google_transit_2018_18_08.zip], which returns empty. 

I have compared my local repo with remote master, now it was updated. I will change back to stable rust version and try import again.

Thanks for your kindly reply and time.
